server/channels/api4/system.go

```diff
@@ -281,7 +281,7 @@ func getAudits(c *Context, w http.ResponseWriter, r *http.Request) {
 	audit.AddEventParameter(auditRec, "audits_per_page", c.Params.LogsPerPage)
 
 	if err := json.NewEncoder(w).Encode(audits); err != nil {
-		c.Logger.Warn("Error while writing response", mlog.Err(err))
+		c.AppContext.Logger().Warn("Error while writing response")
```

Comment:
While there is no different between using the two loggers, `c.Logger` is the one that is used consistently. 
```suggestion
		c.Logger.Warn("Error while writing response")
```